conventions/README.md

```diff
@@ -503,3 +504,11 @@ const elements = this.el.shadowRoot ? this.el.shadowRoot.querySelector("slot").a
 ```
 
 To ensure that all components are compatible for prerendering a prerender build is done as part of `npm test`.
+
+## Writing Tests
+
+### Prevent logging unnecessary messaging in the build
+
+When writing tests, prevent emitting console warnings by stubbing them.
+
+Console warnings can end up polluting the build output messaging that makes it more difficult to identify real issues. Instead of emitting `console.warn()`, be sure to stub `console.warn` in the tests in order to prevent them from displaying in the build.
```

Comment:
Small suggestion: 

> Instead of emitting `console.warn()`, be sure to stub `console.warn`

to

> By stubbing `console.warn`, you can prevent warning messages from displaying in the build. See [`calcite-color.e2e`](https://github.com/Esri/calcite-components/blob/af0c6cb/src/components/calcite-color/calcite-color.e2e.ts#L9-L17) for an example.